### DISCUSSION TITLE
blog/first-day-out

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#11] [Q&A]
# blog/first-day-out

Ottenuto il badge, sono finalmente uscito di casa. Prima giornata alla NASA.

https://www.pompelmo.xyz/blog/first-day-out

**Comment by peppocola** [comment]
grandissimo Giorgione

**Comment by hellisabet** [comment]
AMC>>>>NASA
Recensisci nelle storie Fleabag e Split.
